content/contributors/projects/_index.md

```diff
@@ -735,7 +735,19 @@ Sandbox Projects
  -   **Developer List/Forum:** [Kyverno Discussions](https://github.com/kyverno/kyverno/discussions)
  -   **License:** [Apache 2.0](https://github.com/kyverno/kyverno/blob/main/LICENSE)
  -   **Legal Requirements:** [DCO](https://developercertificate.org/)
- ---
+---
+
+### OpenFunction
+
+ *"OpenFunction is a cloud-native open source FaaS (Function as a Service) platform aiming to let you focus on your business logic without having to maintain the underlying runtime environment and infrastructure. You only need to submit business-related source code in the form of functions."* - [OpenFunction](https://openfunction.dev/)
+
+ -   **Project Repository:** https://github.com/OpenFunction/OpenFunction
+ -   **Contributor Guide:** [Development Guide](https://github.com/OpenFunction/OpenFunction/blob/main/docs/development/README.md)
+ -   **Chat:** Slack: `#openfunction` in [cloud-native.slack.com](https://cloud-native.slack.com)
```

Comment:
[cloud-native.slack.com](https://cloud-native.slack.com)
=>
[slack.cncf.io](https://slack.cncf.io/ )

just like harbor
![image](https://user-images.githubusercontent.com/18525465/167646871-42cd675f-70ea-43e9-b479-b62c3b00dd72.png)

Reply:
We need to fix this and then the conflicts

Reply:
ok